blocks/note/note.css

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+.note {
+  border: 2px solid #707070;
+  border-radius: 4px;
+  color: #707070;
```

Comment:
Could we please use CSS variables here as they could be reused in other components as well

Reply:
done in [0f40da4](https://github.com/adobecom/helpx-internal/pull/3/commits/0f40da49ae8316b958698d5db6f0ffde24f496ae)